php/class-ee.php

```diff
@@ -803,21 +803,24 @@ public static function error_to_string( $errors ) {
 	/**
 	 * Launch an arbitrary external process that takes over I/O.
 	 *
-	 * @access public
+	 * @access   public
 	 * @category Execution
 	 *
-	 * @param string $command External process to launch.
-	 * @param boolean $exit_on_error Whether to exit if the command returns an elevated return code.
+	 * @param string  $command         External process to launch.
+	 * @param boolean $exit_on_error   Whether to exit if the command returns an elevated return code.
 	 * @param boolean $return_detailed Whether to return an exit status (default) or detailed execution results.
+	 * @param array   $env             Environment variables to set when running the command.
+	 * @param string  $cwd             Directory to execute the command in.
+	 *
 	 * @return int|ProcessRun The command exit status, or a ProcessRun object for full details.
 	 */
-	public static function launch( $command, $exit_on_error = true, $return_detailed = false ) {
+	public static function launch( $command, $exit_on_error = true, $return_detailed = false, $env = array(), $cwd = null ) {
 		Utils\check_proc_available( 'launch' );
```

Comment:
Default should be null here. This would ensure that process getting created gets same variable as the process that created it.